net/CMakeLists.txt

```diff
@@ -94,15 +94,11 @@ add_library(HSPlasmaNet
             ${PN_GATE_SOURCES}  ${PN_GATE_HEADERS}
             ${PN_SOURCES}       ${PN_HEADERS}
 )
-target_link_libraries(HSPlasmaNet HSPlasma ${STRING_THEORY_LIBRARIES})
-if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "SunOS")
-    target_link_libraries(HSPlasmaNet ${OPENSSL_LIBRARIES})
-endif()
-
-if(WIN32)
-    target_link_libraries(HSPlasmaNet crypt32)
-    target_link_libraries(HSPlasmaNet ws2_32)
-endif(WIN32)
+target_link_libraries(HSPlasmaNet
+                      HSPlasma string_theory
+                      $<$<NOT:$<PLATFORM_ID:SunOS>>:OpenSSL::Crypto>
+                      $<$<PLATFORM_ID:Windows>:ws2_32>
```

Comment:
What happened to `crypt32`?  Is it not needed?

Reply:
Correct, `OpenSSL::Crypto` correctly links all of OpenSSL's transitive dependencies. This is a major benefit to using targets, where available 😄